code/ship/ship.h

```diff
@@ -1507,6 +1507,7 @@ extern void ship_get_eye( vec3d *eye_pos, matrix *eye_orient, object *obj, bool
 extern ship_subsys *ship_get_indexed_subsys( ship *sp, int index, vec3d *attacker_pos = NULL );	// returns index'th subsystem of this ship
 extern int ship_get_index_from_subsys(ship_subsys *ssp, int objnum, int error_bypass = 0);
 extern int ship_get_subsys_index(ship *sp, const char* ss_name, int error_bypass = 0);		// returns numerical index in linked list of subsystems
+extern int ship_get_subsys_index(ship *shipp, ship_subsys *subsys, int error_bypass = 0);
```

Comment:
Since `error_bypass` is used as a boolean variable it should be a `bool`.

Reply:
See previous comment re matching the other functions.